docs/data_management/res/cheaha_individual_account.csv

```diff
@@ -0,0 +1,6 @@
+Permissions/Responsibilities,Research Computing,User, Storage Owner's Supervisor
+Create personal Cheaha account,,Yes ✔️,
+Move unused data to LTS or archive,,Yes ✔️,
+Manage backup plans,,Yes ✔️,
+Data and access control,,Yes ✔️,
```

Comment:
For line 5, let's add a "With Security Exception" under "Supervisor" column, with an asterisk `*` symbol.

Then we add a footnote (prefixed with an asterisk `*`) under the table describing the circumstances where a supervisor can gain access. We can cross-link to the "Security exception" section.